manifests/charts/gateways/istio-ingress/values.yaml

```diff
@@ -185,6 +185,11 @@ global:
   imagePullSecrets: []
   # - private-registry-key
 
+  # Lifecycle for istio-proxy container; allows you to add preStop/postStop for
+  # istio-proxy container for istio-ingressgateway.
+  istioproxy:
```

Comment:
Should we add this into the https://istio.io/latest/docs/reference/config/istio.operator.v1alpha1/#KubernetesResourcesSpec @ostromart ?